### PR TITLE
Ixia support for custom init container

### PR DESCRIPTION
**Pull request by anjan-keysight**
Ixia support for custom init container through KNE config files

### CODE REVIEW

File: topo/node/ixia/ixia.go [hunk 1]
```diff
@@ -59,6 +59,11 @@ func (n *Node) newCRD() *ixiatg.IxiaTG {
 		},
 	}
 
+	if n.GetProto().Config != nil {
+		ixiaCRD.Spec.InitContainer.Image = n.GetProto().Config.InitImage
+		ixiaCRD.Spec.InitContainer.Sleep = n.GetProto().Config.Sleep
```
Comment (alexmasi) [inline topo/node/ixia/ixia.go:64]: sleep should be the sleep time before starting the pod according to the documentation: https://github.com/openconfig/kne/blob/1775c426748ac4100544b08c9383709da53593fe/proto/topo.proto#L125

Does this instead create the pod without sleeping but then start the initcontainer after the sleep period?

**Review by alexmasi — DISMISSED** [review]

**Comment by alexmasi** [comment]
Looks like there are some merge conflicts, once those are fixed I can merge

**Comment by anjan-keysight** [comment]
Resolved go.sum conflict.

**Comment by alexmasi** [comment]
there are still merge conflicts unfortunately

**Comment by anjan-keysight** [comment]
I have merged main and resolved the issue. Once this is merged I will sync the other PR and resolve conflicts.

**Review by alexmasi — APPROVED** [review]

**Pull request merged**